tour/ranges-splits.md

```diff
@@ -2,22 +2,22 @@
 title: Ranges and Splits
 ---
 
-A [Range] is a specified group of Keys. There are many different ways to create a Range.  Here are a few examples:
+A [Range] is a specified group of keys. There are many different ways to create a `Range`.  Here are a few examples:
 
 ```java
 Range r1 = new Range(startKey, endKey);  // Creates a range from startKey inclusive to endKey inclusive.
 Range r2 = new Range(row);               // Creates a range that covers an entire row.
 Range r3 = new Range(startRow, endRow);  // Creates a range from startRow inclusive to endRow inclusive.
 ```
 
-A Scanner by default will scan all Keys in a table but this can be inefficient. It is a good practice to set a range on a Scanner.
+A `Scanner` by default will scan all `Keys` in a table but this can be inefficient. It is a good practice to set a range on a `Scanner`.
 
 ```java
 scanner.setRange(new Range("id0000", "id0010"));  // returns rows from id0000 to id0010
 ```
 
-As your data grows larger, Accumulo will split tables into smaller pieces called Tablets.  Tablets can then be distributed across multiple Tablet Servers.  
-By default, a table will get split into Tablets on row boundaries, guaranteeing an entire row to be on one Tablet Server.  We have the ability to 
+As your data grows larger, Accumulo will split tables into smaller pieces called `Tablets`.  `Tablets` can then be distributed across multiple `Tablet Servers`.  
+By default, a table will get split into `Tablets` on row boundaries, guaranteeing an entire row to be on one `Tablet Server`.  We have the ability to 
```

Comment:
The idea I had for highlighting was to avoid including the pluralizing `s` inside the highlighted portion, as in:

```suggestion
By default, a table will get split into `Tablet`s on row boundaries, guaranteeing an entire row to be on one `Tablet Server`.  We have the ability to 
```

That way, it's clear that the `s` isn't part of the class name and exists only to pluralize.